integration-tests/stable/dbschema/default.esdl

```diff
@@ -8,6 +8,14 @@ module default {
 
   type User {
     identity: ext::auth::Identity;
+
+    multi movies: Movie;
+    multi shows: Show;
+
+    multi watching_list := ( 
+      select .movies union .shows
```

Comment:
Can we add a third item to the union, to test that we properly union more than two types. And if edgedb allows it, can it also not inherit from `Content` but have at least one of same fields to check we're handling common fields not just common ancestors?

Reply:
hmm how do u mean common fields? if they have the same field but that field is not inherited, it is not "common" imo

Reply:
Yes, you probably shouldn't write an actual schema like that, and I'm not 100% sure off the top of my head if edgedb supports it (I think it does), but if it does the querybuilder should still handle that just in case.